### ISSUE TITLE
What to do for a supporter seeing "BLOCK Archiver Status" CRITICAL

### ISSUE THREAD

**Issue by ChrisM-S** [ISISComputingGroup/IBEX#2145]
As a support person, I would like to know what to do when I see the above message.  I understand that the status associated is CRITCAL and assume this means that beam-time may be lost already (or will be very soon) so I must act right away, possibly by instructing someone over the phone.  I would like to know

- [ ] What specifically can I do to fix this message quickly so beam-time is no longer at risk?

- [ ] If this is not possible, are there specific programs, or files or other diagnostics to direct someone to, or which, if I can log in I may be able use to get a better diagnosis?

- [ ] What steps can I do to reset/revert to a system version which is known not to produce this error.

- [ ] For this error maybe it points to (say) a failure in some electronics, who would be the best person to call up if it cannot be solved - what information could I give to them to help. what might I be able to do to help them.

These may not be the best questions or all of them, but for the purpose of this ticket, the focus is on resolution and the assumption is that this error is worth being alerted to because something can be done.  If there is another error which occurs along with this, it may be that this error is not of use to support staff at all but another one is more pertinent.  Consideration should be given to which one is key and which others should just be logged for diagnosis.

**Comment by FreddieAkeroyd** [comment]
I agree that some procedures need to be documented. If you are referring to IRIS in this particular case, then it looks like they have started to run with SECI - unfortunately nagios doesn't know about this.
